internal/internal_workflow_testsuite_test.go

```diff
@@ -1247,3 +1247,124 @@ func (s *WorkflowTestSuiteUnitTest) Test_WorkflowLocalActivityWithMockAndListene
 	s.NoError(err)
 	s.Equal("hello mock", result)
 }
+
+func (s *WorkflowTestSuiteUnitTest) Test_SignalChildWorkflow() {
+	// This test will send signal from parent to child, and then child will send back signal to ack. No mock is needed.
+	signalName := "test-signal-name"
+	signalData := "test-signal-data"
+	childWorkflowFn := func(ctx Context, parentExec WorkflowExecution) (string, error) {
```

Comment:
Shouldn't parent ids be in the context?

Reply:
no, it is not there, server has that info, but it is not sending down in the DecisionTask.
Task created: https://github.com/uber/cadence/issues/540

Reply:
In the test environment, yes, all info are available. But in production environment, it is not.